Makefile

```diff
@@ -19,6 +19,8 @@ PUSH_ASSETS ?= false
 LOCAL_DETACHED ?= false
 LOCAL_DOCKER_COMPOSE_CONFIG = docker-compose-local.yml
 
+export RUBY_VERSION
```

Comment:
The `RUBY_VERSION` environment variable is exported here in the Makefile using
`RUBY_VERSION := $(shell cat .ruby-version)`.
Alternatively, we could export
`WEB_BASE_DOCKERFILE_FROM ?= ruby:$(RUBY_VERSION)-slim-bullseye`
and use it if we wanted to control the full base image here instead of just the Ruby version.